src/vehicle.cpp

```diff
@@ -3391,20 +3398,22 @@ void vehicle::noise_and_smoke( double load, double time )
 {
     const std::array<int, 8> sound_levels = {{ 0, 15, 30, 60, 100, 140, 180, INT_MAX }};
     const std::array<std::string, 8> sound_msgs = {{
-        "", _("hummm!"), _("whirrr!"), _("vroom!"), _("roarrr!"), _("ROARRR!"),
-        _("BRRROARRR!"), _("BRUMBRUMBRUMBRUM!")
-    }};
+            "", _( "hummm!" ), _( "whirrr!" ), _( "vroom!" ), _( "roarrr!" ), _( "ROARRR!" ),
+            _( "BRRROARRR!" ), _( "BRUMBRUMBRUMBRUM!" )
+        }
+    };
     double noise = 0.0;
     double mufflesmoke = 0.0;
     double muffle = 1.0;
     double m = 0.0;
     int exhaust_part = -1;
     for( size_t p = 0; p < parts.size(); p++ ) {
-        if( part_flag(p, "MUFFLER") ) {
-            m = 1.0 - (1.0 - part_info(p).bonus / 100.0) * double( parts[p].hp() ) / part_info(p).durability;
+        if( part_flag( p, "MUFFLER" ) ) {
+            m = 1.0 - ( 1.0 - part_info( p ).bonus / 100.0 ) * double( parts[p].hp() ) / part_info(
```

Comment:
Maybe move the whole `part_info( p ).durability` to the next line?

Reply:
Refactored the entire concept.
Fixed.